### ISSUE TITLE
Doesnt handle hot water sensor (breaks) and has some other error handling concerns

### ISSUE THREAD

**Issue by xAPPO** [ralfvd/nl.bevlogenheid.evohome#8]
If you have the evoHome hot water sensor unfortunately this breaks - it doesn't find any thermostats.
Essentially the hot water sensor has a blank name and different settable values.

I posted more info in the evoHome thread on the Homey forum
http://forum.athom.com/discussion/comment/57376/#Comment_57376

Not sure if this is being maintained ???

**Comment by Swiftnesses** [comment]
Ralf, 

I'm about to install my water sensor, can you perhaps try and see if there's a workaround we can use? 👍

**Comment by ralfvd** [comment]
I can check, but not having a water sensor myself AND Evohome not having a documented API , it is a bit difficult to adjust the code for this. 

Is one of you capable of getting the status calls and getting the XML output that the system generates?

Alternatively, you could grant access for me to your system via an extra user (unfortunately, there is no 'read-only' access). That way I might be able to getting the XML output of your system myself and see if it's easy to adjust the code for
1) at least ignore the water sensor, so you can install the thermostats
2) build support for the water sensor

**Comment by Swiftnesses** [comment]
@xAPPO as you have a live water thermostat, is there a chance you can do this? BTW, I'd love to see the water temperature logged to insights! (if possible).

**Comment by xAPPO** [comment]
I can do that ..   I'll set you up as an extra user.. will do that tomorrow

**Comment by Swiftnesses** [comment]
@xAPPO Hi Kevin, does Ralf now have access?

**Comment by xAPPO** [comment]
I have set up an account and will email the details to Ralph shortly (I've got your email address)

**Comment by ralfvd** [comment]
Access has been granted and I think I figured out a way to at least ignore the hot water device. I've asked Kevin to test.  In parallel, I need to see if I can whip up a driver for the hot water device. That will take more time than this -hopefully- quick and dirty fix.

**Comment by Swiftnesses** [comment]
A driver would be AMAZING, we've had issues with out hot water (hence the new stat) and I';d love to track it's heat properties.

**Comment by ralfvd** [comment]
1.0.4 which has better selection for thermostats so it won't break when using domestic hot water sensors, has been released in the Athom app store. let me know how it works for you!

**Comment by ralfvd** [comment]
As V2 is now in stable, I'll close this issue. Please create a new issue if you encounter any issue
